src/service/validation/validation.marker.service.spec.ts

```diff
@@ -1,24 +1,23 @@
 import { async, TestBed, inject } from '@angular/core/testing';
-import { HttpModule, XHRBackend, Response, ResponseOptions } from '@angular/http';
-import { MockBackend, MockConnection } from '@angular/http/testing';
-import { AuthConfig, AuthHttp } from 'angular2-jwt';
 import { XtextValidationMarkerServiceConfig } from './xtext.validation.marker.service.config';
 import { ElementType, WorkspaceElement } from '@testeditor/workspace-navigator';
 import { ValidationMarkerService, ValidationSummary } from './validation.marker.service';
-import { XtextNaiveValidationMarkerService } from './xtext.naive.validation.marker.service';
-
-const sampleResponseBody = '{"issues":[\
-    {"description":"WebBrowser cannot be resolved.","severity":"error","line":19,"column":13,"offset":553,"length":10},\
-    {"description":"WebBrowser cannot be resolved.","severity":"error","line":33,"column":13,"offset":925,"length":10},\
-    {"description":"component/mask is not defined in aml","severity":"warning","line":19,"column":13,"offset":553,"length":10},\
-    {"description":"test step could not resolve fixture","severity":"info","line":20,"column":4,"offset":567,"length":5},\
-    {"description":"No ComponentElement found.","severity":"error","line":20,"column":10,"offset":573,"length":9},\
-    {"description":"test step could not resolve fixture","severity":"info","line":21,"column":4,"offset":586,"length":6},\
-    {"description":"test step could not resolve fixture","severity":"info","line":25,"column":4,"offset":709,"length":4},\
-    {"description":"test step could not resolve fixture","severity":"info","line":29,"column":12,"offset":815,"length":3},\
-    {"description":"component/mask is not defined in aml","severity":"warning","line":33,"column":13,"offset":925,"length":10},\
-    {"description":"test step could not resolve fixture","severity":"info","line":34,"column":4,"offset":939,"length":4},\
-    {"description":"test step could not resolve fixture","severity":"info","line":35,"column":4,"offset":961,"length":5}]}';
+import { HttpClient, HttpClientModule } from '@angular/common/http';
+import { HttpClientTestingModule, HttpTestingController } from '@angular/common/http/testing';
+import { XtextDefaultValidationMarkerService } from './xtext.default.validation.marker.service';
+
+// const sampleResponseBody = '{"issues":[\
```

Comment:
can / should this stuff be removed?

Reply:
Yes it can be removed, left it there as a reminder (for you|koray) to write some tests for this service ;-)